content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -138,18 +144,20 @@ To use this approach:
               protocol: TCP
     ```
 
-2. Configure the kubelet on your selected node to use this web manifest by running it with `--manifest-url=<manifest-url>`. On Fedora, edit `/etc/kubernetes/kubelet` to include this line:
+2. Configure the kubelet on your selected node to use this web manifest by
```

Comment:
```suggestion
1. Configure the kubelet on your selected node to use this web manifest by
```

Reply:
@tengqm you missed this one

Reply:
To be honest, this numbering issue is not relevant to the current PR.
I'd more inclined to have a PR self-contained and focused.
With that, I'm happy to fix the numbering issue in a new commit.